cms/templates/js/add-xblock-component-button.underscore

```diff
@@ -1,8 +1,8 @@
 <% if (type === 'advanced' || templates.length > 1) { %>
-<a href="#" class="multiple-templates add-xblock-component-button" data-type="<%= type %>">
+<button type="button" class="multiple-templates add-xblock-component-button" data-type="<%= type %>">
 <% } else { %>
-<a href="#" class="single-template add-xblock-component-button" data-type="<%= type %>" data-category="<%= templates[0].category %>">
+<button type="button" class="single-template add-xblock-component-button" data-type="<%= type %>" data-category="<%= templates[0].category %>">
```

Comment:
Same here.  (It looks like the `add-xblock-component-button` is already being used for the styling of this, so I left this as is.)